pipelines/tests/karate/karateTestsScheduledPipeline.groovy

```diff
@@ -76,18 +74,19 @@ pipeline {
             steps {
                 script {
                     def jobParameters = [
-                        string(name: 'branch', value: params.branch),
-                        string(name: 'threadsCount', value: "4"),
-                        string(name: 'modules', value: ""),
-                        string(name: 'okapiUrl', value: okapiUrl),
-                        string(name: 'tenant', value: 'supertenant'),
-                        string(name: 'adminUserName', value: 'super_admin'),
-                        password(name: 'adminPassword', value: 'admin'),
-                        string(name: 'prototypeTenant', value: prototypeTenant)
+                        branch         : params.branch,
+                        threadsCount   : "4",
+                        modules        : "",
+                        okapiUrl       : okapiUrl,
+                        edgeUrl        : edgeUrl,
+                        tenant         : 'supertenant',
+                        adminUserName  : 'super_admin',
+                        adminPassword  : 'admin',
+                        prototypeTenant: prototypeTenant
                     ]
 
                     sleep time: 60, unit: 'MINUTES'
-                    karateTestsJob = build job: karateTestsJobName, parameters: jobParameters, wait: true, propagate: false
+                    karateStages(jobParameters)
```

Comment:
Same as in Vitaly PR. What do you think about karateFlow name for such type of function?

Reply:
yes, sounds better. Chnaged